src/components/articles/ArticlesPage.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <section class="articles-page">
+    <div class="pa-2 articles-widget__slider__item" v-for="(item, index) in items" :key="index">
+      <articles-widget-item :item="item" />
```

Comment:
Item is an object containing `{ loading, article }`, I think it would be more clear if we pass the two props separately. Something [like this](https://github.com/criticat-team/criticat-client/compare/v2...v2-suggestion). What do you think?

Reply:
Completely agree, I tend to make one-prop components despite knowing it's not good.